types/office-js-preview/index.d.ts

```diff
@@ -102503,904 +102442,3925 @@ declare namespace PowerPoint {
         load(propertyNamesAndPaths?: {
             select?: string;
             expand?: string;
-        }): PowerPoint.Tag;
+        }): PowerPoint.BulletFormat;
         /**
         * Overrides the JavaScript `toJSON()` method in order to provide more useful output when an API object is passed to `JSON.stringify()`. (`JSON.stringify`, in turn, calls the `toJSON` method of the object that is passed to it.)
-        * Whereas the original PowerPoint.Tag object is an API object, the `toJSON` method returns a plain JavaScript object (typed as `PowerPoint.Interfaces.TagData`) that contains shallow copies of any loaded child properties from the original object.
+        * Whereas the original PowerPoint.BulletFormat object is an API object, the `toJSON` method returns a plain JavaScript object (typed as `PowerPoint.Interfaces.BulletFormatData`) that contains shallow copies of any loaded child properties from the original object.
         */
-        toJSON(): PowerPoint.Interfaces.TagData;
+        toJSON(): PowerPoint.Interfaces.BulletFormatData;
     }
     /**
      *
-     * Represents the collection of tags.
+     * Specifies the connector type for line shapes.
      *
      * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
      * @beta
      */
-    class TagCollection extends OfficeExtension.ClientObject {
-        /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
-        context: RequestContext;
-        /** Gets the loaded child items in this collection. */
-        readonly items: PowerPoint.Tag[];
+    enum ConnectorType {
         /**
-         * Adds a new tag at the end of the collection. If the `key` already exists in the collection, the value of the existing tag will be replaced with the given `value`.
-         *
-         * [Api set: PowerPointApi BETA (PREVIEW ONLY)]
-         * @beta
+         * Straight connector type
```

Comment:
A lot of these enum field descriptions are missing periods. Could we do a pass on the source file and make sure these descriptions include the proper punctuation? Some seem a bit brief too (though maybe more detail isn't needed).

Reply:
@AlexJerabek These shape APIs are copied from Excel. The PPT team has documented them better than in Excel, which often have no description at all. Here's the Excel version of this enum:

![image](https://user-images.githubusercontent.com/9088056/134093157-ed2b12ea-f50f-4956-9764-b72fba6d9a2b.png)

If we want all these better described, how about we make that a long term work item instead of holding up PowerPoint now?

Reply:
@Rick-Kirkham, sounds good. I didn't mean to hold up this check in. I meant to imply we should spend time in the future fixing this. I'll create a work item for this and the Excel Shape APIs (which I assume will share most of the documentation).